packages/edit-post/src/components/block-manager/index.js

```diff
@@ -63,6 +64,13 @@ function BlockManager( {
 						),
 						numberOfHiddenBlocks
 					) }
+					<Button
+						variant="secondary"
```

Comment:
```suggestion
						variant="tertiary"
```

I think the `tertiary` button style might be more appropriate:

<img width="657" alt="image" src="https://github.com/WordPress/gutenberg/assets/36432/121d399b-6087-4ce4-89cb-19ace98cb286">